static/templates/subscription.handlebars

```diff
@@ -90,6 +92,17 @@
               <input type="submit" name="change-stream-description" value="{{t 'Change stream description' }}" class="btn btn-danger change-stream-description-button" tabindex="-1" />
             </form>
           </div>
+          <div class="change-stream-mandatory-status">
+            <button class="btn btn-danger toggle-stream-mandatory" data-stream-name="{{name}}" tabindex="-1">
+              {{#if mandatory}}
+              {{t "Change stream mandatory status to false"}}
```

Comment:
This needs to be changed at all. You probably want to refactor this code so that the same string isn't present in 2 locations